### PR TITLE
feat(cli): adds status cmd to fetch block height

### LINKED ISSUE #107: Add to the validator CLI the ability to return status
Cosmovisor currently read from the story cli to try to get status of the current block height. But `story` doesn't support the `status` subcommand yet

This is required to support our hardfork using cosmovisor

### PR DESCRIPTION
**Pull request by leeren**
adds validator CLI subcommand for returning block height

issue: fixes #107

### CODE REVIEW

**Review by limengformal — APPROVED** [review]

File: client/cmd/validator.go [hunk 1]
```diff
@@ -553,6 +574,58 @@ func unstakeOnBehalf(ctx context.Context, cfg stakeConfig) error {
...
 }
 
+func checkStatus(ctx context.Context, cfg baseConfig) error {
+	req, err := http.NewRequestWithContext(ctx, http.MethodGet, cfg.RPC+"/status", nil)
+	if err != nil {
+		return errors.Wrap(err, "failed to create request")
+	}
+
+	// Create an HTTP client and perform the request
+	client := &http.Client{}
```
[the remark this answers is not in the document]
Comment (leeren) [reply client/cmd/validator.go:584]: Added

**Review by edisonz0718 — CHANGES_REQUESTED** [review]
Need to confirm if this is how cosmovisor use it

File: client/cmd/validator.go [hunk 2]
```diff
@@ -553,6 +575,58 @@ func unstakeOnBehalf(ctx context.Context, cfg stakeConfig) error {
...
+	if err != nil {
+		return errors.Wrap(err, "failed to query cometBFT status endpoint")
+	}
+	defer resp.Body.Close()
+
+	if resp.StatusCode != http.StatusOK {
+		return errors.New("unexpected response code: " + strconv.Itoa(resp.StatusCode))
+	}
+
+	type statusResponse struct {
```
[the remark this answers is not in the document]
Comment (leeren) [reply client/cmd/validator.go]: Updated to use the RPC client!

**Comment by leeren** [comment]
> Need to confirm if this is how cosmovisor use it

Confirmed to work on cosmovisor after simulating a local upgrade

**Review by limengformal — APPROVED** [review]

**Pull request merged**